### PR TITLE
Package Java and CSharp on linux GHA

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: .github/actions/package_csharp/scripts/mac_build_csharp.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+BUILD_EXAMPLES:BOOL=ON
+BUILD_TESTING:BOOL=ON
+SimpleITK_BUILD_DISTRIBUTE:BOOL=ON
+SimpleITK_BUILD_STRIP:BOOL=1
+EOM
+
+export CTEST_CACHE
+export CTEST_BINARY_DIRECTORY="${GITHUB_WORKSPACE}/CSharp"
+
+javac -version
```
Comment (zivy) [inline .github/actions/package_csharp/scripts/mac_build_csharp.sh:25]: Checking the java compiler version is likely a remnant of the java build script which isn't relevant for the C# script here. If observation is correct, please remove.

File: .github/workflows/Package.yml [hunk 2]
```diff
@@ -54,9 +58,10 @@ jobs:
       - name: Upload Wheels
```
Comment (zivy) [inline .github/workflows/Package.yml:58]: Please change the step name to "Upload artifacts" or "Upload packages".

File: .github/workflows/Package.yml [hunk 3]
```diff
@@ -187,6 +202,7 @@ jobs:
...
         uses: actions/upload-artifact@v4
```
Comment (zivy) [inline .github/workflows/Package.yml:203]: Please change step name, same as above.

**Review by zivy — COMMENTED** [review]
Some minor changes requested.

**Review by dave3d — APPROVED** [review]

File: .github/actions/package_csharp/scripts/mac_build_csharp.sh (hunk 1 above)
Comment (blowekamp) [reply .github/actions/package_csharp/scripts/mac_build_csharp.sh:25]: done

File: .github/workflows/Package.yml (hunk 3 above)
Comment (blowekamp) [reply .github/workflows/Package.yml:203]: done

**Pull request merged**